packages/components/src/utils/browsers.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { css } from 'emotion';
+
+/* eslint-disable jsdoc/no-undefined-types */
+/**
+ * @param {TemplateStringsArray} strings
+ * @param  {import('create-emotion').Interpolation[]} interpolations
+ */
+export function ieOnly( strings, ...interpolations ) {
```

Comment:
With the recent changes in dropping IE support, is this still needed?

Reply:
Ah I hadn't considered that, you're right!